release-notes/con-relnotes-notable-features.adoc

```diff
@@ -3,143 +3,129 @@
 
-|Catalog common|`@backstage/plugin-catalog-common`|1.0.17|Continuous deployment
+=== RBAC Plugin
+In the new backend system, automatically getting the IDs of all dynamically installed plugins is not possible. This means that permissions of dynamic plugins supporting RBAC will not be automatically displayed in the RBAC UI.
+As a workaround, you can manually add a list of dynamic plugin IDs supporting RBAC in the {product-short} application configuration file.
 
-|Catalog Graph|`@backstage/plugin-catalog-graph`|0.2.37|Continuous deployment
 
-|Catalog Import|`@backstage/plugin-catalog-import`|0.10.1|Continuous deployment
+== Support for Elastic Kubernetes Services (EKS)
 
-|Catalog React|`@backstage/plugin-catalog-react`|1.8.5|Continuous deployment
+You can now integrate your {product} application with Amazon Web Services (AWS), which optimizes your development workflows within the AWS ecosystem.
+To enable this integration, deploy {product-short} in Elastic Kubernetes Services (EKS) using Helm Chart or Operator. 
 
-|Events backend|`@backstage/plugin-events-backend`|0.2.15|Continuous deployment
 
-|Events node|`@backstage/plugin-events-node`|0.2.15|Continuous deployment
+== Support for Azure Kubernetes Services (AKS)
 
-|Home|`@backstage/plugin-home`|0.5.9|Continuous deployment
+You can now seamlessly integrate Azure Kubernetes Services into your development workflow in {product}. Integrating {product-short} with AKS can simplify the deployment, management, and scaling of the application on Microsoft Azure’s fully managed Kubernetes service. 
 
-|Kubernetes backend|`@backstage/plugin-kubernetes-backend`|0.13.0|Container management
 
-|Org|`@backstage/plugin-org`|0.6.15|Identity and access management
+== Support for viewing installed plugins using Web UI
 
-|Permission backend|`@backstage/plugin-permission-backend`|0.5.29|Identity and access management
+You can now use {product-short} web interface (Web UI) to view a table of plugins that are installed. This feature uses the `dynamic-plugins-info` frontend component, which offers a table of plugins that are currently installed in {product} or Janus IDP. You can apply client-side sorting, filtering, and pagination on the plugins table.
 
-|Permission common|`@backstage/plugin-permission-common`|0.7.9|Identity and access management
 
-|Permission node|`@backstage/plugin-permission-node`|0.7.17|Identity and access management
+== Supported plugins in {product} {product-version}
 
-|Permission react|`@backstage/plugin-permission-react`|0.4.16|Identity and access management
+The following table lists the supported plugins in {product} {product-version}:
 
-|Proxy backend|`@backstage/plugin-proxy-backend`|0.4.4|Continuous deployment
+[id="supported-plugins-in-rhdh"]
+.Supported plugins in {product} {product-version}
+[cols="30%,30%,10%,30%",options="header",subs="+attributes"]
+|===
+| Plugin name | Plugin ID | Version | Category
```

Comment:
"Category" column probably doesn't make sense if it is only in one plugin

Reply:
rather than having the list of plugins in a **THIRD** place, maybe we could just link to the main table in ref-rhdh-plugins.adoc ? https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/75/files#diff-48d14956b347986734708cd4e5df54eadbc40ea3c686e2b1b2e0088fb3f2a655

Reply:
I agree with Nick's suggestion. better to provide a link to that table here. Doing the same.